library/src/main/java/io/constructor/core/ConstructorIo.kt

```diff
@@ -1123,6 +1124,48 @@ object ConstructorIo {
         )
     }
 
+    /**
+     * Tracks generic result click events
+     * ##Example
+     * ```
+     * ConstructorIo.trackGenericResultClick("Pencil", "123", "234")
+     * ```
+     * @param itemName the name of the item clicked
+     * @param customerId the id of the item clicked
+     * @param variationId the variationId of the item clicked
+     * @param sectionName section of the item clicked
+     */
+    fun trackGenericResultClick(itemName: String, customerId: String, variationId: String? = null, sectionName: String? = null) {
+        var completable = trackItemDetailLoadedInternal(itemName, customerId, variationId, sectionName)
```

Comment:
Should this be `trackGenericResultClickInternal` instead?

Reply:
Correct 👍